src/traits.rs

```diff
@@ -249,6 +250,10 @@ where
 }
 
 /// A distributed algorithm that defines a message flow.
+///
+/// Many algorithms require a source of random data, which must be supplied on each call. It is up
```

Comment:
"source of random data" could be just "RNG" for definiteness.

Reply:
:+1: